setup.py

```diff
@@ -37,45 +37,40 @@ def _get_version():
     raise ValueError('`__version__` not defined in `deepchem/__init__.py`')
 
 
-setup(
-    name='deepchem',
-    version=_get_version(),
-    url='https://github.com/deepchem/deepchem',
-    maintainer='DeepChem contributors',
-    classifiers=[
-        'Development Status :: 4 - Beta',
-        'Environment :: Console',
-        'Intended Audience :: Developers',
-        'Intended Audience :: Information Technology',
-        'License :: OSI Approved :: MIT License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python :: 3.7',
-        'Programming Language :: Python :: 3.8',
-        'Programming Language :: Python :: 3.9',
-    ],
-    license='MIT',
-    description='Deep learning models for drug discovery, \
+setup(name='deepchem',
```

Comment:
This file should not be modified. We should add this as a soft dependency and not a hard dependency.

Ideally, can we avoid use of einops? Adding an extra dependency to deepchem is always extra maintenance burden. einops is fairly popular so it may be OK, but good to see if we can avoid it

Reply:
If a new soft dependency is added, it will need to be added to the docs/ folder as well

Reply:
[`rearrange()`](https://einops.rocks/api/rearrange/) seems to be nothing more than a different API for basic numpy operations like transpose and reshape.  We should be able to easily do the same thing with numpy calls.  It's not worth adding a new dependency for that.